doc/tor.md

```diff
@@ -18,7 +26,7 @@ information in the debug log about your Tor configuration.
 CLI `-addrinfo` returns the number of addresses known to your node per network
 type, including Tor v2 and v3. This is useful to see how many onion addresses
```

Comment:
Since #22050, and for Bitcoin Core 22.0, our node shouldn't know of Torv2 addresses. So this should be updated. 

```suggestion
type. This is useful to see how many onion addresses
```

```
$ ./src/bitcoin-cli -addrinfo

{
  "addresses_known": {
    "ipv4": 11975,
    "ipv6": 2395,
    "torv2": 0,
    "torv3": 10,
    "i2p": 0,
    "total": 14380
  }
}
```

Reply:
Hm! There is the same info in the release notes for -addrinfo. For now, -addrinfo returns "torv2" and "torv3", so my thought was that this should be updated when -addrinfo is changed to return only "onion" instead. WDYT?

Reply:
yeah, makes sense. Can be left for a followup